Meetings.md

```diff
@@ -1,6 +1,14 @@
 # Stack Club Meetings
 
-Individual session links and recordings are given below, most recent meeting at the top.
+Individual session links and recordings are given below, most recent meeting at the top.  Note that for now (starting Jan. 2019), we will have a "regular" sessions only every two weeks (where we will show / discuss notebooks), and "hack-only" sessions on off-weeks, where we hope to have a DM Project person present to answer questions and address concerns.  Only the "regular" sessions are announbced below.  
+
+| Session  | Date  | Topic  | Links  |
+|---|---|---|---|
+| Phase 3, Session 1 | Friday February 8, 2019 [(video)]() | Overview of 2 notebooks, hacking | [Project List](Andrew Bradshaw's notebook on Fe55 and overview of Keith Bechtol's "LSST Verification" notebook;  links to be inserted) |
```

Comment:
@Madejski I thought the idea was that Andrew would do the 8th, and Keith would do the 22nd? Happy to pre-approve this PR as-is, but you should check with Keith whether he can do the 22nd. It'd be good to focus on one notebook per session, and also schedule two sessions ahead (so that people can plan).